### PR TITLE
adds buildtoken command (closes #100)

### LINKED ISSUE #100: Add command to get build token
It would be nice to provide a command to access a build token rather than having to login in to the UI.  This would obviously require the user to be authenticated first.

```
$ harbor-compose login
$ harbor-compose buildtoken ls

SHIPMENT  ENVIRONMENT   CICD_ENVAR        TOKEN                             
app1      dev           APP1_DEV_TOKEN    tCMrTbm2P3ofn1CYq3WmTg4po2P2lJdR  
app2      dev           APP2_DEV_TOKEN    bm8tk6aAGam2T4iyWOZN0JgEsysatCmZ
```

In the future we could consider adding a command to roll/rotate the token.

```
$ harbor-compose buildtoken roll
```

Thoughts?

### PR DESCRIPTION
**Pull request by jritsema**
```
Available Commands:
  get         displays a build token for the requested shipment and environment
  list        list harbor build tokens for shipments in harbor-compose.yml

harbor-compose buildtoken ls

SHIPMENT             ENVIRONMENT   CICD_ENVAR                     TOKEN
mss-poc-sqs-web      dev           MSS_POC_SQS_WEB_DEV_TOKEN      3xFVlltLZ7JwPH20Km75DrpMwOk2asdf
mss-poc-sqs-worker   dev           MSS_POC_SQS_WORKER_DEV_TOKEN   2N3QFkQkdilwj34ezS2JTxwt6Fn3asdf


harbor-compose buildtoken get

Shipment: mss-poc-sqs-worker
Environment: dev

SHIPMENT             ENVIRONMENT   CICD_ENVAR                     TOKEN
mss-poc-sqs-worker   dev           MSS_POC_SQS_WORKER_DEV_TOKEN   2N3QFkQkdilwj34ezS2JTxwt6Fn3asdf


harbor-compose buildtoken get mss-poc-sqs-worker dev

SHIPMENT             ENVIRONMENT   CICD_ENVAR                     TOKEN
mss-poc-sqs-worker   dev           MSS_POC_SQS_WORKER_DEV_TOKEN   2N3QFkQkdilwj34ezS2JTxwt6Fn3asdf
```

### CODE REVIEW

**Comment by jritsema** [comment]
Added `-e` flag to list command for parity with deploy/catalog.

```
$ harbor-compose buildtoken ls -e qa

SHIPMENT             ENVIRONMENT   CICD_ENVAR                     TOKEN
mss-poc-sqs-web      qa            MSS_POC_SQS_WEB_DEV_TOKEN      ihtvPrAH84ULVm6IC7LjWvXUgEhr7cnQ
mss-poc-sqs-worker   qa            MSS_POC_SQS_WORKER_DEV_TOKEN   Y3Jk0DmMaUsoWO8mbI2Edn9Ixhwj14Vd
```

**Comment by jkurz** [comment]
The 100th PR. Let's celebrate. :)

I mean it closes #100. Same thing. :)

File: cicd.md [hunk 1]
```diff
@@ -63,7 +72,7 @@ $ harbor-compose generate mss-my-shipment dev --build-provider circleciv1
 Be sure to supply the following environment variables in your Circle CI build:
 DOCKER_USER (registry user)
 DOCKER_PASS (registry password)
-MSS_MY_SHIPMENT_DEV_TOKEN (Harbor shipment/environment build token)
+MSS_MY_SHIPMENT_DEV_TOKEN=2N3QFkQkdilwj34ezS2JKxwt6Fn3yuA7
```
Comment (jkurz) [inline cicd.md:75]: This should be a command and not the actual token.

like 

```
MSS_MY_SHIPMENT_DEV_TOKEN=$(harbor-compose buildtoken ls -e qa)
```

**Review by jkurz — APPROVED** [review]

**Comment by jkurz** [comment]
Code looks good. Just a few comments on the Readme.

**Comment by jkurz** [comment]
yeah I was not thinking straight about my comment on the Readme. Please ignore. The user will have to copy and paste into circleCI no matter what so even if the command did what I was thinking, it would not matter. LGTM

**Pull request merged**